components/BudgetItemsList.js

```diff
@@ -314,7 +315,19 @@ const BudgetItem = ({ item, isInverted, isCompact, canDownloadInvoice, intl }) =
             mt={[2, 0]}
           >
             <Flex alignItems="center" flexWrap="wrap">
-              {description ? (
+              {isExpense ? (
+                <Link route={`/${collective.slug}/expenses/${item.id}`}>
+                  {description ? (
+                    <P color="black.900" fontWeight="600" fontSize="Paragraph" title={description}>
+                      {truncate(description, { length: 60 })}
+                    </P>
+                  ) : (
+                    <P color="black.500" fontStyle="italic" fontWeight="600" fontSize="Paragraph">
+                      <FormattedMessage id="BudgetItemsList.NoDescription" defaultMessage="No description provided" />
+                    </P>
+                  )}
+                </Link>
```

Comment:
That's quite a lot of duplicated code. Can we extract that to a variable above, something like:

```jsx
const formattedDescription = description ? (
  <P color="black.900" fontWeight="600" fontSize="Paragraph" title={description}>
    {truncate(description, { length: 60 })}
  </P>
) : (
  <P color="black.500" fontStyle="italic" fontWeight="600" fontSize="Paragraph">
    <FormattedMessage id="BudgetItemsList.NoDescription" defaultMessage="No description provided" />
  </P>
);

// In render
{isExpense ? <Link>{formattedDescription}</Link> : formattedDescription}
```

Reply:
yes, refactored the logic 👍